ts/src/pro/binance.ts

```diff
@@ -1128,12 +1129,11 @@ export default class binance extends binanceRest {
         client.resolve (newTickers, 'tickers');
     }
 
-    signParams (params = {}) {
-        this.checkRequiredCredentials ();
-        let extendedParams = this.extend ({
-            'timestamp': this.nonce (),
-            'apiKey': this.apiKey,
-        }, params);
+    signParams (isPrivate: boolean, params = {}) {
+        if (isPrivate) {
```

Comment:
also, when this is not private this method is almost useless, so I will restore it and create a different method for the public endpoints if needed